daml-script/daml/Daml/Script.daml

```diff
@@ -92,6 +96,7 @@ query p = Script $ Free $ Query (QueryACS p (templateTypeRep @t) (pure . map (\(
 
 data AllocateParty a = AllocateParty
   { displayName : Text
+  , idHint : Optional Text
```

Comment:
Protobuf and therefore the ledger API doesn’t differentiate between an empty string and a string not being present. You can therefore just remove the `Optional` here.

Reply:
Indeed, done.